scripts/ffcv/create_ffcv_datasets.py

```diff
@@ -140,6 +138,20 @@ def cache_streaming(remote, local, num_workers=64):
         pass
 
 
+class PILImageClassStreamingDataset(StreamingDataset):
+    """
+        A subclass of StreamingDataset that asserts and returns samples that are
+        tuples (PIL.Image, np.int64).
+        Intended to enforce sample types for pipelines that expect the tuple format (image, class)
+    """
+    def __getitem__(self, idx: int) -> Tuple[Image.Image, np.int64]:
+        obj = super().__getitem__(idx)
+        assert 'x', 'y' in obj.keys()
+        assert isinstance(obj['x'], Image.Image)
+        assert isinstance(obj['y'], np.int64)
+        return (obj['x'], obj['y'])
```

Comment:
@dskhudia the FFCV writer, as well as our PIL image collator, expect samples to be tuples not dicts, so I had to add this step here.

Reply:
Looks great. Thank you!